internal/cloud/realm/dependencies_test.go

```diff
@@ -97,7 +113,23 @@ func TestRealmDependencies(t *testing.T) {
 	})
 }
 
-func parseZipArchive(t *testing.T, file *os.File) map[string]string {
+func waitForInstallation(realmClient realm.Client, groupID, appID string) error {
```

Comment:
For testing, this seemed like the best way to go about this to keep `cloud/realm` and `command` parts of the code separate. Not sure if there might be a better approach